crates/iroha_core/src/smartcontracts/wasm.rs

```diff
@@ -1788,160 +1800,4 @@ mod tests {
 
         Ok(())
     }
-
-    #[test]
-    async fn instructions_not_allowed() -> Result<(), Error> {
```

Comment:
the tests were not working correctly and it's very hard to inject failure conditions into iroha

Reply:
because for the test to implement correctly I would have to write inline webassembly that serializes result into linear memory. Current implementation was incorrect because it was missing `drop`